graphql2/isotimestamp.go

```diff
@@ -13,10 +14,14 @@ import (
 func MarshalISOTimestamp(t time.Time) graphql.Marshaler {
 	return graphql.WriterFunc(func(w io.Writer) {
 		if t.IsZero() {
-			io.WriteString(w, "null")
+			if _, err := io.WriteString(w, "null"); err != nil {
+				log.Println("ERROR: Issue with MarshalISOTimestamp when using io.WriteString:", err)
+			}
 			return
 		}
-		io.WriteString(w, `"`+t.UTC().Format(time.RFC3339Nano)+`"`)
+		if _, err := io.WriteString(w, `"`+t.UTC().Format(time.RFC3339Nano)+`"`); err != nil {
+			log.Println("ERROR: Issue with MarshalISOTimestamp when using io.WriteString:", err)
+		}
 	})
```

Comment:
changed to ignore the error since I personally don't think we need the error message, and if you're ok with ignoring it in these cases where we don't really care if it errors anyways.

Reply:
Yeah, I'm reasonably sure gqlgen builds this in a buffer before sending, so no error should be possible. If it is, it would be because someone closed a tab, which wouldn't be helpful.